package.json

```diff
@@ -28,7 +28,7 @@
     "test:unit": "jest"
   },
   "dependencies": {
-    "sort-package-json": "1.46.1"
+    "sort-package-json": "cvent/sort-package-json#fields-option"
```

Comment:
I'll update this if/when the https://github.com/keithamus/sort-package-json/pull/206 PR gets merged